File: react/src/swagger/api/models/LessonStudentResponseModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type LessonStudentResponseModel = {
    id?: string;
    studentId?: string;
    studentUserName?: string | null;
    lessonId?: string;
    lessonName?: string | null;
    midterm?: number | null;
    final?: number | null;
    makeup?: number | null;
};

File: aspnet-core/Jormungandr/Jormungandr.Application/Models/LessonStudent/LessonStudentResponseModel.cs
namespace Jormungandr.Application.Models.LessonStudent;
public class LessonStudentResponseModel : BaseResponseModel
{
    public Guid StudentId { get; set; }
    public string StudentUserName { get; set; }
    public Guid LessonId { get; set; }
    public string LessonName { get; set; }
    public int? Midterm { get; set; }
    public int? Final { get; set; }
    public int? Makeup { get; set; }
}
